jobs/e2e_node/crio/crio_evented_pleg.ign

```diff
@@ -9,17 +9,27 @@
   },
   "storage": {
```

Comment:
@haircommander @harche as discussed: We now only change this job before rolling out the other configurations.

We now use a drop-in configuration for CRI-O directly (`40-evented-pleg.conf`) without having a need for setting the `Environment=ENABLE_POD_EVENTS` (cc @sairameshv)

Reply:
/lgtm